sdk/core/azure_core_test/src/lib.rs

```diff
@@ -20,20 +23,35 @@ const SPAN_TARGET: &str = "test-proxy";
 /// to setup up the HTTP client to record or play back session records.
 #[derive(Debug)]
 pub struct TestContext {
+    repo_dir: &'static Path,
     crate_dir: &'static Path,
+    service_directory: &'static str,
```

Comment:
Is `service_directory` a folder or a path?
Should it be `service_folder`? If using directory then `service_dir` for consistency.